src/runtime_src/core/common/shim/buffer_handle.h

```diff
@@ -87,6 +89,18 @@ class buffer_handle
   bind_at(size_t /*pos*/, const buffer_handle* /*bh*/, size_t /*offset*/, size_t /*size*/)
   {
   }
+
+  virtual void
+  sync_aie_bo(xrt::bo&, const char *, xclBOSyncDirection, size_t, size_t)
```

Comment:
Try to remove xrt::bo object from this. We shouldn't  require this.

Reply:
bo instance is getting used in many "Aie" functions, we can cleanup with separate PR